### PR TITLE
Hassan/feat/splash screen

### PR DESCRIPTION
**Pull request by muhammadhassan2k3**
splash screen is the screen that is shown while our app loads.

* Added splash screen
* eslint configurations updated
* navigation version 6 added

### CODE REVIEW

File: src/screens/HomeScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react-native/no-inline-styles */
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+
+function HomeScreen() {
+  return (
+    <View style={styles.containerStyle}>
+      <Text style={{ fontSize: 50, marginTop: 60 }}>HomeScreen</Text>
```
Comment (KamranKhankhail) [inline src/screens/HomeScreen.tsx]: Can you please move inline style recommended seperate styles.

**Review by KamranKhankhail — CHANGES_REQUESTED** [review]
Mostly looks good.

I've added a comment kinldy check & let me know if any question is there.

File: src/screens/HomeScreen.tsx (hunk 1 above)
Comment (muhammadhassan2k3) [reply src/screens/HomeScreen.tsx]: okay sure, let me do.

**Review by KamranKhankhail — APPROVED** [review]
LGTM. 👍🏼

**Pull request merged**